### PR TITLE
Avoid generate rulesets.json in makecrx.sh, and refine comments

### PR DESCRIPTION
**Pull request by yfdyh000**
"crx" uses `default.rulesets`.

### CODE REVIEW

File: makecrx.sh [hunk 1]
```diff
@@ -33,25 +33,11 @@ fi
...
-# implementing this with make, but make is very slow with 15k+ input files.
-needs_update() {
-  find src/chrome/content/rules/ -newer $RULESETS_JSON |\
-    grep -q .
-}
-if [ ! -f "$RULESETS_JSON" ] || needs_update ; then
-  echo "Generating ruleset DB"
-  python2.7 ./utils/make-json.py && bash utils/validate.sh && cp pkg/rulesets.json src/chrome/content/rulesets.json
-fi
-
```
Comment (Hainish) [inline makecrx.sh]: We need this for the build process.

File: makecrx.sh [hunk 2]
```diff
@@ -33,25 +33,11 @@ fi
 
 VERSION=`python2.7 -c "import json ; print(json.loads(open('chromium/manifest.json').read())['version'])"`
 
-echo "Building chrome version" $VERSION
+echo "Building crx version" $VERSION
```
Comment (Hainish) [inline makecrx.sh]: This comment has been updated in https://github.com/EFForg/https-everywhere/tree/webextensions already.

File: utils/make-json.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python2.7
 #
 # Builds a JSON DB containing all the rulesets, indexed by target.
-# The JSON DB is used by the Firefox addon.
+# The JSON DB is used by the Chrome extension & Firefox (WebExtension) addon.
```
Comment (Hainish) [inline utils/make-json.py]: I'll add this comment in https://github.com/EFForg/https-everywhere/tree/webextensions

File: makecrx.sh (hunk 1 above)
Comment (yfdyh000) [inline makecrx.sh]: @Hainish Can you explain why we need it? Thank you.

**Pull request closed without merging**